CHARTER.md

```diff
@@ -0,0 +1,232 @@
+# JSON Schema Charter
+<!-- This document is managed in the json-schema-org/community GitHub repository. Please do NOT modify this file in another repository as changes may be overridden. -->
+
+## 0: Guiding Principles
+The JSON Schema project is part of the OpenJS Foundation. The JSON Schema project strives to be open and transparent as much as is possible, and wish to enable anyone to interact and engage with any area of our work.
+
+Having no structure in place usually leads to one that is informal and undocumented, making it difficult to meet our own expectations of how we wish to operate. As such, we define the following charter which includes aspects of the governance model to which we subscribe and by which we operate.
+
+## 1: Scope
+JSON Schema aims to enable the confident and reliable use of the JSON data format. It does this primarily by providing specification documents which define a declarative language that allows annotation and validation of JSON documents.
+While JSON Schema's primary target is constraint-based data validation, it continues to be used across the whole stack, in different stages of a given project, and for purposes beyond its original design. We aim to enable these additional and emergent use cases.
+
+### 1.1: In-scope
```

Comment:
Should we highlight interoperability as an "in-scope" item?  Specifically, if a behavior is intended to only operate in a confined and controlled environment by a single implementation, is that in scope for us?

Reply:
This took me more thinking time than anticipated.
To me, being a "standard" _implies_ interoeprability is of paramount concern. I can't imagine how a standard would not also mean being concerned about interoperability. (I guess I could, maybe, if I thought long enough...)

Under "Publication of the JSON Schema standard", I had included "extensibility". What does that mean? It means that as part of the standard, we should be concerned about extensibility. It matters.

Could "interoperability" go there also? If it did, what would it imply? I don't really know, given, to me, "standard" already covers that.

Then I thought, it could go in the "out of scope" section.

Looking at the comment you linked to, it sounds like you're suggesting that standardising an approach to SVA's is something you'd consider out of scope?

I agree that we shouldn't define anything for a single implementation, but I also think that standardising a way for schema authors to create a two party agreement SVA has value, for things like tooling, an should be considered a good interoperable approach.

Reply:
> Looking at the comment you linked to, it sounds like you're suggesting that standardising an approach to SVA's is something you'd consider out of scope?

No. That comment, and the entire conversation, is about ad-hoc non-SVA keywords.

Reply:
Ah, I see that clearly now. I was... a little tired last time.

OK, so I agree, we should add "interoperability" to "in-scope".
But I think what you're really asking for is to add "non-standard multi-party agreements" to "out of scope", right?

But, we do care about that, because that's what custom SVA would allow.
So, if I'm understanding right, that would mean you're asking for out of scope to include "non-standard multi-party agreements beyond custom single value annotations".

However, vocabularies could be just known between two parties. But you could say that's a "standard" approach.

All this to say, I agree interoperability should be included, but I'm not convinced that covers what you're asking for.

If I'm way off base here, feel free to say so 😅

Reply:
https://github.com/json-schema-org/community/pull/325/commits/aea0b80d98991b81d7c34c2875899cd05bf9f270 FYI